### ISSUE TITLE
Access Denied error on DiagnosticTool

### ISSUE THREAD

**Issue by marcobonafede** [Azure/RDS-Templates#384]
Hi,
I was able to deploy diagnostic tool correctly, but when I try to search for activities (specifying UPN) I got the following error:

Error Code : 404
Error : Access Denied! You are not authorized to get list of activities.Please contact system administrator

I confirm that I'm RDS Owner of the tenant that I created.
In general I'm global admin on azure subscription where resources were created.

Thanks

**Comment by robinschaerer** [comment]
Hi,
I'm experiencing the same error.
I have confirmed that the account is (inherited) owner of the log analytics workspace and has the role "RDS Owner" on the rds tenant.

The resources were only 3 or 4 hours ago deployed, so maybe there is just not any data in the log analytics workspace yet, so I will check back in some time again. But I will also follow this issue for any other news.

Thanks in advance for any advice!

**Comment by marcobonafede** [comment]
> Hi,
> I'm experiencing the same error.
> I have confirmed that the account is (inherited) owner of the log analytics workspace and has the role "RDS Owner" on the rds tenant.
> 
> The resources were only 3 or 4 hours ago deployed, so maybe there is just not any data in the log analytics workspace yet, so I will check back in some time again. But I will also follow this issue for any other news.
> 
> Thanks in advance for any advice!

My resources were deployed 2 days ago but I'm still facing this issue

**Comment by geek2point0** [comment]
I too am facing this issue. I am an Owner on the subscription, the app does also have the Contributor role on the subscription and it deployed without issue. Happy to provide an information I can do troubleshoot.

**Comment by albandrod** [comment]
Me too, I have followed all the steps in:

https://docs.microsoft.com/en-us/azure/virtual-desktop/deploy-diagnostics and usediagnostics

but I'm receiving the error:

Error Code : 404
Error : Access Denied! You are not authorized to get list of activities.Please contact system administrator

**Comment by lintFan** [comment]
I wonder if this could be due to the note of the docs https://docs.microsoft.com/en-us/azure/virtual-desktop/deploy-diagnostics  here that says "As of March 16, 2020, we've temporarily disabled diagnostic queries that impacted user experience due to increased demand on the service. This will cause the tool to stop working because it relies on those queries to function. We'll update this article when diagnostic queries are available again."
@ChristianMontoya  or @stgeorgi  could that be why they are getting this error?

**Comment by stgeorgi** [comment]
yes, its impacting our management portal as it uses the same functionality

**Comment by martinyy108** [comment]
Same here and I have issue open as well

**Comment by marcobonafede** [comment]
does anyone found a resolution?

**Comment by stgeorgi** [comment]
lintFan shared this earlier "I wonder if this could be due to the note on the docs https://docs.microsoft.com/en-us/azure/virtual-desktop/deploy-diagnostics that says "As of March 16, 2020, we've temporarily disabled diagnostic queries that impacted user experience due to increased demand on the service. This will cause the tool to stop working because it relies on those queries to function. We'll update this article when diagnostic queries are available again."

**Comment by lintFan** [comment]
It looks like the URL for that documentation was moved to here https://docs.microsoft.com/en-us/azure/virtual-desktop/virtual-desktop-fall-2019/deploy-diagnostics
